### PR TITLE
fixes #929 - Added a config setting to allow setting the default diff type

### LINKED ISSUE #929: Can't change default diff to side by side
Need to have a config setting to allow setting the default diff type.

### PR DESCRIPTION
**Pull request by srfrnk**
Updated the readme as well.

### CODE REVIEW

**Comment by FredrikNoren** [comment]
Nice. Make sure to bump the package version and add a line to the changelog as well. See https://github.com/FredrikNoren/ungit/blob/master/CONTRIBUTING.md for details

**Comment by srfrnk** [comment]
done.

File: README.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ Example of `~/.ungitrc` configuration file to change default port and enable bug
 ```json
 {
 	"port": 8080,
-	"bugtracking": true
+	"bugtracking": true,
+	"diffType": "textdiff" | "sidebysidediff"
```
Comment (codingtwinky) [inline README.md]: An example in readme should be valid and this is a simple small example of the rc file.  Can we please revert this change?

File: components/textdiff/textdiff.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ var Type = function() {
   var textDiff = 'textdiff'
 
   this.text = ko.observable("Default");
-  this.value = ko.observable(textDiff);
+  this.value = ko.observable(ungit.config.diffType||textDiff);
```
Comment (codingtwinky) [inline components/textdiff/textdiff.js]: in `.../source/config.js` can we add this variable along with standard commenting?

Also can we add quick validation where if wrong value is passed in, default to text diff with a warning log?

I think that would make this change more useable

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ Use the following format for additions: ` - VERSION: [feature/patch (if applicab
 - 1.1.21:
```
Comment (FredrikNoren) [inline CHANGELOG.md]: Sorry I just noticed the version in package.json and changelog.md was out of sync. Fixed on master now. This one should be 1.1.22 instead.

Comment (codingtwinky) [inline CHANGELOG.md]: Sorry, that is my fault...

File: README.md (hunk 1 above)
Comment (srfrnk) [inline README.md]: Sure I just thought since there is no other place to show the options this would be the best place - WDYT?

File: components/textdiff/textdiff.js (hunk 2 above)
Comment (srfrnk) [inline components/textdiff/textdiff.js]: Sure - but I can't see any way to show a warning log. console.log is not displayed since it's a forked process - am I wrong?

**Comment by srfrnk** [comment]
All change requests have been resolved - any chance you merge the pull request?

**Comment by FredrikNoren** [comment]
@srfrnk I pushed the missing fixes for you: https://github.com/FredrikNoren/ungit/pull/934/commits/662ddd8b3cb37b871838d093aa87209164d2fed1 This is the new PR with the fixes: https://github.com/FredrikNoren/ungit/pull/934
